cli/script/command-executor.ts

```diff
@@ -178,6 +178,24 @@ function appList(command: cli.IAppListCommand): Promise<void> {
         });
 }
 
+function checkIfAppExists(appName: string): Promise<App> {
+    log(chalk.cyan(`Checking if app with the name "${appName}" exists...\n`));
```

Comment:
I don't think we need this line, as it's taken for granted that we will check the app name (all of the other CLI's effectively do the same but don't log this step unless there is an error).